sandbox/descriptions.html.example

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
```

Comment:
@heff I can do this, although it took me a while to realize that HTML in the docs/examples directory (including the existing `simple-embed/index.html`) don't work in that directory, only once they've been copied into the dist directory, because of the need for the built video.js files. Might it be worth having a note to that effect somewhere, since the HTML files in the sandbox work where they are?